packages/waku/src/adapters/cloudflare-build-enhancer.ts

```diff
@@ -31,18 +40,25 @@ export default {
       `\
 {
   "name": "waku-project",
```

Comment:
If this `"name"` is important, can the value be taken from package.json's name field?